nixos/modules/services/networking/bitlbee.nix

```diff
@@ -33,7 +33,7 @@ let
 
   purple_plugin_path =
     lib.concatMapStringsSep ":"
-      (plugin: "${plugin}/lib/pidgin/")
+      (plugin: "${plugin}/lib/purple-2/")
```

Comment:
Does that work for all relevant plugins? For instance, `purple-matrix` and `purple-hangouts` use `lib/pidgin`. 
But I don't know if these are supposed to be used with bitlbee, I don't use it myself.

Reply:
well the majority of them use now purple-2, at least all I checked. I guess, for compatibility we can also include both.